### PR TITLE
Add devel to CI

### PR DESCRIPTION
**Pull request by 0atman**
As per #89

### CODE REVIEW

File: inim.nimble [hunk 1]
```diff
@@ -9,7 +9,7 @@ bin           = @["inim"]
 
 # Dependencies
 
-requires "nim >= 1.0.0"
+#requires "nim >= 1.0.0" # can we remove this to imply it should work with all versions?
```
Comment (timotheecour) [inline inim.nimble:12]: is inim working with nim < 1.0.0 ? if not then you should keep this

Comment (0atman) [reply inim.nimble:12]: @timotheecour thanks Tim, but the test fails on devel with this line, how can I get round this?

Comment (0atman) [reply inim.nimble:12]: Here's the error using `nimlang/nim:devel` and the require nim 1 line:
```
$ nimble build
Unsatisfied dependency: nim (>= 1.0.0)
```

Comment (timotheecour) [reply inim.nimble:12]: you may need to track down which version of nim is installed that leads to this failure.
try:

run: nimble build
=>
run: nim -v && nimble build --verbose

or something similar to get more debug info

Comment (0atman) [reply inim.nimble:12]: Much obliged @timotheecour , trying now.

Comment (0atman) [reply inim.nimble:12]: `Nim Compiler Version 0.15.3 (2016-12-13) [Linux: amd64]`

Fun! I wonder why the docker container isn't using the latest. Must be the wrong tag, I'll see what the right one is.

**Comment by 0atman** [comment]
using choosenim for devel requires recompiling each test - what was a 20s test is now 5 minutes :cry:

**Comment by 0atman** [comment]
@timotheecour are you aware of a workaround for compiling our own version of nim? the `devel` tag on the nim docker image hasn't been built for years

**Comment by 0atman** [comment]
Phew, that was not fun.

**Pull request merged**